ppdc/ppdc-catalog.cxx

```diff
@@ -63,8 +63,8 @@ ppdcCatalog::ppdcCatalog(const char *l,	// I - Locale
 #if defined(__APPLE__) && defined(CUPS_BUNDLEDIR)
     char		applelang[256];	// Apple language ID
     CFURLRef		url;		// URL to cups.strings file
-    CFReadStreamRef	stream = NULL;	// File stream
-    CFPropertyListRef	plist = NULL;	// Localization file
+    CFReadStreamRef	stream = nullptr;	// File stream
```

Comment:
Aside from my prior comments, CoreFoundation is a C API and CANNOT use any such C++ constructs (`nullptr` is NOT the same as `NULL` in some cases...)